tests/no-applyshim/custom-style-only.html

```diff
@@ -0,0 +1,72 @@
+<!doctype html>
```

Comment:
This should have scoping shim and test scoping.

Reply:
Done